README.md

```diff
@@ -13,6 +13,30 @@ Currently documentation is sparse. Please help us out!
 
 Get [docker](http://docker.com).
 
+Quick overview of Docker can be found:
+[Docker Quick Start Guide](https://docs.docker.com/engine/quickstart/)
+
+First you must install and configure the database component [faf-db](https://github.com/FAForever/db)
+You can either manually install the component and follow the instructions on the Github page or run the script - init_and_wait_for_db.sh (Linux and MAC only)
+In order to run the script,, make it executable by running
+
+    chmod +x init_and_wait_for_db.sh
```

Comment:
Already did it here https://github.com/FAForever/api/commit/9f74e4610ae34e17f8ee5c93d2bf5bb8100d0167